### DISCUSSION TITLE
Header Fehler im Themes Timeline ...

### DISCUSSION THREAD

**Discussion by ElfenTroll** [ophian/styx#22] [General]
Hallo Ian,

Du hattest mich ja auf dieses "Forum" verwiesen, wenn es sich nicht um Bugs und so handelt. Ich möchte gerne noch einmal auf mein "Headerproblem" zurückkommen, weil da immer noch etwas nicht stimmt.

In den Einstellungen für den Header im Themes Timeline kann man ja zwei getrennte Headerbilder angeben. Das scheint aber definitiv nicht zu funktionieren, was ich schade finde!
Wenn ich für den unteren Headerbereich (Subheader) ein Bild eingebe, wird es immer über beide Container insgesamt angezeigt. Gebe ich ein Bild für den oberen Container (Kopfzeilenbild) an, so wird es gar nicht angezeigt. Ich wollte eigentlich den oberen Headerbereich ohne Bild und den unteren mit Hintergrundbild.

Hast Du eine Idee dazu, bzw. Ahnung, wo der Fehler liegen könnte, und könntest Du mir eine Korrektur geben. Das mit der Bildgröße habe ich übrigens noch gelesen. Danke!

Gruß Michael E.

![h1](https://user-images.githubusercontent.com/101506660/158078667-346a1096-0bf0-41f4-8781-d580f14d3e78.png)
![h2](https://user-images.githubusercontent.com/101506660/158078668-c2e71c3c-1e59-4356-9255-fe0b6824a21a.png)

**Comment by ophian** [comment]
Laut der Beschreibung handelt es sich bei dem **Subheader** um das eigentliche header Hintergrundbild, so wie du es hast.
Das **header** Bild ist als solches etwas missverständlich, denn es handelt sich laut Titel um ein "Optionales Kopfzeilenbild. Leer für Blognamen", das folgender Beschreibung entspricht: "Empfohlene Größe des Logo Header-Bildes: 150 x 40px. Wird nur angezeigt, wenn "' . USE_CORENAV . '" auf "'. YES .'" steht". Für mich heißt das, es ersetzt das "BLOGGERnotizen & MINDENnotizen" H1 Element (und zwar nur das) durch ein entsprechendes Bild, und zwar, wenn die Option USE_CORENAV aktiviert ist. Ist das bei dir der Fall?

**Comment by ElfenTroll** [comment]
Hi,

also ich wüsste nicht, dass ich die Funktion "USE_CORENAV" aktiviert habe bzw. wo sie aktiviert wird! Die Einzige von mir vorgenommenen Änderung war bisher die mit dem Umbruch des Namens in der Handyansicht via user.css ... Wo finde ich die Funktion den, oder in welcher Datei kann ich das kontrolllieren?

 Im "Original" von Timeline war es halt so dass der Hintergrund des Blognamens dunkel war und das Bild eben nur darunter im Cointainter eingefügt wurde (Subheader) ... Das sah eigentlich recht gut aus, weil man dann auch den Blognamen gut lesen konnte, und da möchte ich wieder hin.

Hast Du noch eine Idee dazu?

**Comment by ophian** [comment]
In der timeline Theme Konfiguration in der Klappbox **Navigations-Optionen** gibt es diese Option:
**Globale Navigation verwenden?** Ja Nein
Das hättest du bestimmt selbst gefunden, wenn du die Description info Box wirklich gelesen hättest (ich habe nur aus der lang Datei zitiert. USE_CORENAV ist selbst eine lang Konstante).

Damit ich wirklich verstehe ob es sich eventuell um einen Fehler in der Logik handelt, muss ich wissen, ob das so wie programmiert auch passiert oder nicht.

**Comment by ElfenTroll** [comment]
MuHa Ian,

wie sagt man: Meine Schuld und Asche auf mein Haupt ... Habe es jetzt gefunden! Es hing zusammen mit Navigationslinks im Header und jetzt passt es, und die Suche ist auch da .... SORRY nochmals, bin halt echt auf dem Gebiet komplett Laie und da übersieht man schon mal etwas. Thema kann geschlossen werden! :-) :-)
